deps.ffmpeg/50-libtheora.zsh

```diff
@@ -0,0 +1,163 @@
+autoload -Uz log_debug log_error log_info log_status log_output
+
+## Dependency Information
+local name='libtheora'
+local -A versions=(
+  macos 1.1.1
+  linux 1.1.1
+  windows 1.1.1
```

Comment:
Does this mean we're building libtheora for Windows even if we're not using it in FFmpeg yet? I'd prefer if we just didn't build it for Windows at all until a later commit or PR. You could even stash the changes away and make a separate PR right after this one, and that would be fine with me. To me, it makes more sense that way, and also would provide great examples for how to add functionality to the build scripts.

Reply:
It's excluded by the `targets` variable below, the build script ignores any target not contained in it, _if_ it's specified. So in effect the file is entirely skipped right now.

Reply:
I suppose that's fine. We'll just have to specify the version number for Windows whenever we do enable it.